src/main/java/org/jenkinsci/plugins/workflow/libs/LibraryCachingCleanup.java

```diff
@@ -8,29 +8,47 @@
 import java.io.IOException;
 import java.util.concurrent.TimeUnit;
 
+import jenkins.util.SystemProperties;
+
 @Extension public class LibraryCachingCleanup extends AsyncPeriodicWork {
-    private final Long recurrencePeriod;
-    private final Long unreadCacheClearTime;
+    public static /* non-final for script console */ int EXPIRE_AFTER_READ_DAYS =
+            SystemProperties.getInteger(LibraryCachingCleanup.class.getName() + ".CACHE_EXPIRY_DAYS", 7);
 
     public LibraryCachingCleanup() {
         super("LibraryCachingCleanup");
-        recurrencePeriod = Long.getLong("jenkins.workflow-libs.cacheCleanupPeriodDays", TimeUnit.DAYS.toMillis(7));
-        unreadCacheClearTime = Long.getLong("jenkins.workflow-libs.unreadCacheClearDays", TimeUnit.DAYS.toMillis(7));
     }
 
     @Override public long getRecurrencePeriod() {
-        return recurrencePeriod;
+        return TimeUnit.HOURS.toMillis(12);
```

Comment:
Previously this was 7 days, which combined with `unreadCacheClearTime` being 7 days meant that in the worst case a library could go almost 2 weeks without being cleaned up, and if Jenkins restarts in that time frame, the timer would start all over again.

Since the check is not that expensive, I think it is fine to run it twice a day.